addons/product/models/product_pricelist_item.py

```diff
@@ -112,10 +124,16 @@ def _default_pricelist_id(self):
              "Rounding is applied after the discount and before the surcharge.\n"
              "To have prices that end in 9.99, set rounding 10, surcharge -0.01")
     price_surcharge = fields.Float(
-        string="Price Surcharge",
+        string="Extra Fee",
         digits='Product Price',
         help="Specify the fixed amount to add or subtract (if negative) to the amount calculated with the discount.")
 
+    price_discount_surcharge = fields.Float(
```

Comment:
does it happen to use both `price_surcharge` and `price_discount_surcharge` or could we use one field for the two logic ?